### DISCUSSION TITLE
Saral App Usage Observation: Configurable App Icon

### DISCUSSION THREAD

**Discussion by karsood** [Sunbird-Saral/Community#10] [Ideas]
Observation: The app currently has a blue colour icon which is not configurable

Impact: Since there will be multiple states with their own Saral apps, it is important for there to be a differentiation in the app icon.

Desire: States to configure their App Icon.

**Comment by surendrasinghs** [comment]
@dileep-gadiraju this looks like an obvious one. Is this not already possible?

**Reply by dileep-gadiraju** [reply]
@surendrasinghs, App icon goes with apk packaging. So we can add documentation on how to replace the default  App icon while building the same. This has to be done by the implementation team.
@navin9584 / @apurvabisht97 / @arman-tdev ,  Please work on https://github.com/Sunbird-Saral/Community/issues/14 to address the same.
